pkg/swagger/schema.go

```diff
@@ -0,0 +1,74 @@
+package swagger
```

Comment:
Should this be delegated to something like https://github.com/getkin/kin-openapi for parsing and managing OpenAPI documents ?

Reply:
I considered that, even wrote a POC with kin-openapi (nice lib!) this morning. Eventually decided to focus on CRD parsing with the apiextensions.k8s.io itself as I can marshal CRDs directly into structs with that.

The original swagger/openapiv2 parsing could probably use a refactor too, I left the swagger name as it is a bit out of scope for this change right now. Thanks for pointing out that it is trademarked, more the reason to refactor this part.